src/processing/resources/index.js

```diff
@@ -9,6 +9,9 @@ import { ensureTrailingSlash } from '../../utils/url';
 
 function getResourceUrlReplacer (ctx) {
     return function (resourceUrl, resourceType, charsetAttrValue, baseUrl) {
+        if (!urlUtil.isSupportedProtocol(resourceUrl))
```

Comment:
### Algorithm
If we support the protocol then we should proxy url. If not then we shoul kept as is.
see client implementation - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/url.js#L10

On server side exception was raised if protocol is not supported.
So in https://github.com/DevExpress/testcafe-hammerhead/pull/466 we remove error generation for non-supported protocol.
After then we became on server side calculate proxy url for all protocols.

### Regression test
I extend the existing test `Should process stylesheets` (see https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/server/proxy-test.js#L491).
It use two files that was changed:
test/server/data/stylesheet/expected.css
test/server/data/stylesheet/src.css